src/components/AdminDashboard/adminSection.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+import { Link } from 'react-router-dom';
+
+const AdminSection = (props) => {
+  const {
+    title,
+    linkTo,
+    createLinkTo,
+    config: {
+      isLoading,
+      hasError,
+      error,
+      resultCount,
+    },
+  } = props;
+
+  return (
+    <div>
+      <h3>{title}</h3>
+      {
+        isLoading ?
+          <a className="disabled">{title} are loading...</a>
+          :
+          hasError ?
+            <a className="disabled">{error}</a>
+            :
+            resultCount === 0 ?
+              <a className="disabled">No {title} created</a>
+              :
+              <Link to={linkTo}>View All {title} ({resultCount})</Link>
```

Comment:
Nested ternaries like this really aren't that desirable. A better solution may be to split this body into its own component that returns content based on props.

Reply:
This has been updated to hopefully be more legable